pkg/mcs/resource_manager/client/config.go

```diff
@@ -21,11 +21,10 @@ import (
 )
 
 var (
-	requestUnitList map[rmpb.RequestUnitType]struct{} = map[rmpb.RequestUnitType]struct{}{
-		rmpb.RequestUnitType_RRU: {},
-		rmpb.RequestUnitType_WRU: {},
+	requestUnitLimitTypeList map[rmpb.RequestUnitType]struct{} = map[rmpb.RequestUnitType]struct{}{
```

Comment:
Do we still need this `map` now? If not we can remove `for range` where it is called.

Reply:
Since the proto keeps `RequestUnitType` (though it only has 1 type), so I think it's reasonable to keep a list here also.